power-platform/admin/managed-environment-sharing-limits.md

```diff
@@ -39,6 +39,44 @@ If a user tries to share a canvas app that contradicts the sharing rules, they w
 
 :::image type="content" source="media/managed-environment-canvas-app-sharing-rule.png" alt-text="Screenshot of a message when canvas app doesn't respect sharing rule.":::
 
+## Use PowerShell to set sharing limits
+
+You can also use PowerShell to set and remove sharing limits.
+
+### Set sharing limits
+
+Here's an example PowerShell script that excludes sharing with security groups and limits to 20 the number of individuals canvas apps can be shared to, for an existing managed environment:
+
+```powershell
+# Retrieve the environment
+$environment = Get-AdminPowerAppEnvironment -EnvironmentName <EnvironmentId>
+
+# Update the Managed Environment settings
+$governanceConfiguration = $environment.Internal.properties.governanceConfiguration
+$governanceConfiguration.settings.extendedSettings | Add-Member -MemberType NoteProperty -Name 'limitSharingMode' -Value "excludeSharingToSecurityGroups" -Force
+$governanceConfiguration.settings.extendedSettings | Add-Member -MemberType NoteProperty -Name 'maxLimitUserSharing' -Value "20" -Force
+
+# Save the updated Managed Environment settings
+Set-AdminPowerAppEnvironmentGovernanceConfiguration -EnvironmentName <EnvironmentId> -UpdatedGovernanceConfiguration $governanceConfiguration
+```
+
+### Remove sharing limits
+
+Here's an example PowerShell script that removes the sharing limits set by the above script:
```

Comment:
"Here's a PowerShell scripts that removes the sharing limits that were configured in the script above:"